src/community/json-ld/src/main/java/org/geoserver/jsonld/configuration/JsonLdConfiguration.java

```diff
@@ -111,6 +111,10 @@ private NamespaceSupport declareNamespaces(FeatureType type) {
         return namespaceSupport;
     }
 
+    public GeoServerDataDirectory getDd() {
```

Comment:
Ouch, this is really ugly. What is this, data directory? getDataDirectory then. Does it need to be public? I don't see anything else using it, is this a leftover?

Reply:
fixed